src/objects/zcl_abapgit_object_fugr.clas.abap

```diff
@@ -534,9 +537,25 @@ CLASS zcl_abapgit_object_fugr IMPLEMENTATION.
       zcx_abapgit_exception=>raise_t100( ).
     ENDIF.
 
-* The result can also contain function which are lowercase.
+    SELECT * FROM enlfdir
```

Comment:
add a comment that we dont trust the result from RS_FUNCTION_POOL_CONTENTS ? so its double checked...

Reply:
Added comment